source/styleEditor.js

```diff
@@ -1,3 +1,13 @@
-$(document).ready(function(){
-  // your code goes here.
+$(document).ready(function() {
+  $(':submit').on('click', function() {
+    event.preventDefault();
+
+    var form = document.querySelector('form');
```

Comment:
I would be consistent with pure JS vs jQuery. `querySelector` is JS and your `$(selector)...` line is jQuery. Pick one and stick with it.